GVFS/GVFS.Platform.Windows/ProjFSFilter.cs

```diff
@@ -286,6 +281,90 @@ public static bool IsNativeLibInstalled(ITracer tracer, PhysicalFileSystem fileS
             return existsInSystem32 || existsInAppDirectory;
         }
 
+        public static bool TryCopyNativeLibIfDriverVersionsMatch(ITracer tracer, PhysicalFileSystem fileSystem, out string copyNativeDllError)
```

Comment:
I think the logic makes sense here, how were you able to test?

Reply:
I've included some details in the "Manual testing" section of the PR description.  

I did all of the testing on my RS5+ machine, but I manually disabled the inbox ProjFS and then manually installed the ProjFS we package by running `C:\Program Files\GVFS\Filter\prjflt.inf`.

I used the same approach to verify the service catches version mismatches (by using an older version of prjflt.inf/prjflt.sys).